src/module/actor/creature/document.ts

```diff
@@ -406,12 +406,15 @@ abstract class CreaturePF2e<
 
             const updates: (DeepPartial<ArmorSource> & { _id: string })[] = [];
 
-            if (isEquipped(usage, equipped) && item instanceof ArmorPF2e && item.isArmor) {
+            if (isEquipped(usage, equipped) && item.isOfType("armor")) {
                 // see if they have another set of armor equipped
-                const wornArmors = this.itemTypes.armor.filter((a) => a !== item && a.isEquipped && a.isArmor);
+                const wornArmors = this.itemTypes.armor.filter((a) => a !== item && a.isEquipped);
                 for (const armor of wornArmors) {
                     updates.push({ _id: armor.id, system: { equipped: { inSlot: false } } });
                 }
+            } else if (equipped.carryType !== "held" && item.isOfType("weapon") && item.shield?.isRaised) {
+                const raiseAShieldEffect = item.actor.itemTypes.effect.find((e) => e.slug === "effect-raise-a-shield");
```

Comment:
```suggestion
                // Stop raising the shield the weapon is attached to when it becomes stowed
                const raiseAShieldEffect = item.actor.itemTypes.effect.find((e) => e.slug === "effect-raise-a-shield");
```

Took me a bit to realize what this was for :(